clang/lib/AST/ASTStructuralEquivalence.cpp

```diff
@@ -214,6 +214,14 @@ class StmtComparer {
     return E1->size() == E2->size();
   }
 
+  bool IsStmtEquivalent(const DeclRefExpr *DRE1, const DeclRefExpr *DRE2) {
+    if (nullptr == DRE1->getDecl() || nullptr == DRE2->getDecl()) {
+      return false;
+    }
+    return IsStructurallyEquivalent(Context, DRE1->getDecl()->getDeclName(),
```

Comment:
Shouldn't that simply be `IsStructurallyEquivalent(DRE1->getDecl(), DRE2->getDecl())` ?

Reply:
Thanks for review, now I have taken your suggestions.When I develop in llvm 13,```static bool IsStructurallyEquivalent(StructuralEquivalenceContext &Context,Decl *D1, Decl *D2)``` this function do not give me expected result. Maybe now this updated, I will find changes between two versions after